brightcove-exoplayer/OfflinePlaybackSampleApp/src/main/java/com/brightcove/player/samples/offlineplayback/utils/BrightcoveDownloadUtil.java

```diff
@@ -0,0 +1,91 @@
+package com.brightcove.player.samples.offlineplayback.utils;
+
+import android.os.Bundle;
+import android.util.Log;
+
+import com.brightcove.player.model.MediaFormat;
+import com.brightcove.player.model.Video;
+import com.brightcove.player.offline.MediaDownloadable;
+
+import java.util.ArrayList;
+
+/**
+ * A download util class to select audio and caption tracks
+ */
+public class BrightcoveDownloadUtil {
+
+    private static final String TAG = BrightcoveDownloadUtil.class.getSimpleName();
+
+    private BrightcoveDownloadUtil() {
+
+    }
+
+
+    public static void selectMediaFormatTracksAvailable(final Video video, MediaDownloadable mediaDownloadable, Bundle bundle) {
```

Comment:
We should document this method to explain what it is doing. We should note that this here we're arbitrarily selecting certain audio and closed caption tracks for download